log/logger_setup.go

```diff
@@ -30,12 +34,12 @@ func getWriters(s *SubLoggerConfig) (io.Writer, error) {
 		case "stderr":
 			writer = os.Stderr
 		case "file":
-			if FileLoggingConfiguredCorrectly {
-				writer = GlobalLogFile
+			if fileLoggingConfiguredCorrectly {
+				writer = globalLogFile
```

Comment:
This one is a stretch, but, if you have a config with a `file` output, but a nil `fileSettings`, then `fileLoggingConfiguredCorrectly` is false, the `writer` is never set, and subloggers have a `nil` `output *multiWriterHolder`, so a log causes a panic:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x18 pc=0xf3240c]

goroutine 21 [running]:
github.com/thrasher-corp/gocryptotrader/log.loggerWorker()
        C:/Users/gloriousNITS/go/src/github.com/thrasher-corp/gocryptotrader/log/logger_multiwriter.go:46 +0x58c
created by github.com/thrasher-corp/gocryptotrader/log.init.0
        C:/Users/gloriousNITS/go/src/github.com/thrasher-corp/gocryptotrader/log/logger_setup.go:219 +0x25
exit status 2
```
So please put in an `else` which returns an error and that should be the end of it.